src/Model.php

```diff
@@ -21,6 +21,8 @@
 
 /**
  * Data model class.
+ *
+ * @property Field[]|Reference[] $elements
```

Comment:
Is this assumtion right?

Reply:
Currently yes, but in future I think more element types could go there. Maybe even scopes/conditions, joins etc. Not sure, but good for now.